src/rules/noUnboundMethodRule.ts

```diff
@@ -150,3 +225,15 @@ function isSafeUse(node: ts.Node): boolean {
             return false;
     }
 }
+
+function isWhitelisted(node: ts.Node, whitelist: Set<string>, allowTypeof: boolean): boolean {
+    if (isTypeOfExpression(node.parent)) {
+        return allowTypeof;
+    }
+    if (isCallExpression(node.parent)) {
+        const expression = node.parent.expression as ts.Identifier;
+        const callingMethodName = expression.escapedText as string;
```

Comment:
> `expression.escapedText`

Looks like `escapedText` is what's giving you trouble in in TS<=2.4. Switch to `.text` and they seem to work fine: https://circleci.com/gh/JoshuaKGoldberg/tslint/1267

Reply:
Yes, that was the issue. Thanks!